src/main/kotlin/io/github/mojira/arisa/infrastructure/jira/Functions.kt

```diff
@@ -344,6 +344,26 @@ fun addRestrictedComment(
     }
 }
 
+fun addRawComment(
```

Comment:
This is not needed anymore, is it?

Reply:
It'd be nice to leave it for future purposes.